edge-modules/MetricsCollector/Configuration.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace MetricsCollector
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    public class Configuration
```

Comment:
be specific. for example.
```suggestion
    public class MetricsConfiguration
```

Reply:
Bear and I discussed above, but I think the name should indicate that these metrics configurations are sourced from the twin. I have decided to go with MetricsConfigFromTwin

Reply:
If I define default metrics configuration in config file and use it in case these config cannot get from Twin; I should reuse this class, right?  If so, this name is not fit.

Reply:
I'm going to merge this configuration class with Settings.cs. The way this is implemented is counter-intuitive to me. 

I think the only reason it is implemented with two settings/configuration classes is to allow for the endpoints as a dictionary via the twin's desired properties. I had assumed that the GetConfigurationAsync() read from the twin on a timer to refresh the configuration, but that is not the case. The configuration is retrieved once from the twin and never again. It seems counter-intuitive to update the twin and have nothing happen, so I am going to have everything read from the environment.